R/build.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+. /util/utility_functions.sh
+
+if [ -z $2 ] ; then
+    pkg=R
+else
+    pkg=$2
+fi
+
+if [ -z $3 ] ; then
+    version=3.2.0
+else
+    version=$3
+fi
+
+build_deps="libc6-dev zlib1g-dev tcl-dev tk-dev libblas-dev liblapack-dev libcairo2-dev libpng12-dev libtiff4-dev libjpeg-dev libreadline-dev default-jre"
+urls="
+http://cran.rstudio.com/src/base/R-3/$pkg-$version.tar.gz"
```

Comment:
`urls` is technically a list (insofar as a variable value separated by spaces or newlines is a list in Bourne), originally i wrote these as:

```sh
urls="
http://example.org/example.tar.gz
"
```

To make that painfully obvious. It's not necessary to follow suit if there is only one url.